### ISSUE TITLE
How to create a new session

### ISSUE THREAD

**Issue by jasondunsmore** [s-kostyaev/ellama#158]
When I try to create a new session with `C-u M-x ellama-chat`, my prompt is appended to the current session. How do I create a new session (without deleting the current one)?

**Comment by sathlan** [comment]
Hi,

Introduced by https://github.com/s-kostyaev/ellama/commit/4dd5288c05f7103b7947c84694bf1bc47e71dfc9, which is a large commit, so I'm not sure about the right way to solve this.

The "obvious" way is to remove a line from that [block](https://github.com/s-kostyaev/ellama/commit/4dd5288c05f7103b7947c84694bf1bc47e71dfc9#diff-f948c8a67ddb4f98d026d84127dece66d38ee52e16143ed8d97bd581861c25d1R1451) which is what I did locally:

```diff
@@ -1601,7 +1601,6 @@ the full response text when the request completes (with BUFFER current)."
 	 (donecb (plist-get args :on-done))
 	 (provider (if current-prefix-arg
 		       (progn
-			 (setq current-prefix-arg nil)
 			 (eval (alist-get
 				(completing-read "Select model: " variants)
 				providers nil nil #'string=)))
```

Then I was able to have a new session file created by calling: `C-u ellama-chat`

**Comment by s-kostyaev** [comment]
Fixed in 0.12.1
